watchtower-plugin/src/constants.rs

```diff
@@ -44,6 +44,8 @@ pub const RPC_RETRY_TOWER_DESC: &str =
     "Retries to send pending appointment to an unreachable tower";
 pub const RPC_ABANDON_TOWER: &str = "abandontower";
 pub const RPC_ABANDON_TOWER_DESC: &str = "Forgets about a tower and wipes all local data";
+pub const RPC_PING: &str = "ping";
```

Comment:
Rename this to "pingtower" so it does not collide with CLN's existing ping command.

Reply:
oooooo! i see. got it. Thankss